tasks/change-granule-collection-s3/src/index.ts

```diff
@@ -0,0 +1,388 @@
+'use strict';
+
+import { Context } from 'aws-lambda';
+import { runCumulusTask } from '@cumulus/cumulus-message-adapter-js';
+import get from 'lodash/get';
+import keyBy from 'lodash/keyBy';
+import cloneDeep from 'lodash/cloneDeep';
+import zip from 'lodash/zip';
+// eslint-disable-next-line lodash/import-scope
+import { Dictionary } from 'lodash';
+import path from 'path';
+import pMap from 'p-map';
+import { MissingS3FileError, InvalidArgument } from '@cumulus/errors';
+import { S3 } from '@cumulus/aws-client';
+import {
+  unversionFilename,
+} from '@cumulus/ingest/granule';
+import {
+  isCMRFile,
+  granulesToCmrFileObjects,
+  metadataObjectFromCMRFile,
+} from '@cumulus/cmrjs';
+import { BucketsConfig } from '@cumulus/common';
+import { urlPathTemplate } from '@cumulus/ingest/url-path-template';
+import { constructCollectionId } from '@cumulus/message/Collections';
+import { getCollection } from '@cumulus/api-client/collections';
+import { log } from '@cumulus/common';
+import { CollectionRecord } from '@cumulus/types';
+import { CumulusMessage } from '@cumulus/types/message';
+import { CollectionFile } from '@cumulus/types';
+import { BucketsConfigObject } from '@cumulus/common/types';
+import { copyObject } from '@cumulus/aws-client/S3';
+import { getGranule } from '@cumulus/api-client/granules';
+import { getRequiredEnvVar } from '@cumulus/common/env';
+import { fetchDistributionBucketMap } from '@cumulus/distribution-utils';
+import { AssertionError } from 'assert';
+import { apiGranuleRecordIsValid, isCMRMetadataFile, updateCmrFileCollections, uploadCMRFile, ValidApiFile, ValidGranuleRecord } from './update_cmr_file_collection';
+
+const MB = 1024 * 1024;
```

Comment:
Nit: I see we have the pattern elsewhere, but low-key not a fan of replicating this definition everywhere.    We should consider if this is a reasonable case for a common definition/export, but not required in this PR.

Reply:
Nit: not sure exporting it from types is correct either.    This was more noting this should be a constant across *Cumulus*... given we have it defined in at least 3 tasks it seems like it could be DRY-ed up by exporting it from somewhere like common.      Considering this resolved as it's a nit.... but consider pulling it out of types.